frontend/appflowy_flutter/lib/util/text_direction.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:appflowy/workspace/application/settings/appearance/appearance_cubit.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+TextDirection getTextDirectionBaseOnContext(
```

Comment:
I would suggest adding these functions to the `BuildContext` extension to allow easy accessibility.